kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -151,7 +152,9 @@
     protected ObjectSubType financialObjectSubType;
     protected AssetAcquisitionType acquisitionType;
     protected ContractsAndGrantsAgency agency;
+    protected List<Note> boNotes = new ArrayList<Note>();

```

Comment:
There's no mention in the specs for the deletion of notes. As far as deleting Assets? We don't know. You'd have to talk to the BAs about that.